src/mscorlib/src/System/IO/FileSystemInfo.cs

```diff
@@ -185,11 +185,13 @@ public DateTime CreationTime
             }
         
             set {
+#if !FEATURE_CORECLR
```

Comment:
It would be nice to ifdef whole methods not just bodies to make sure that they are not used by accident somewhere.